app/src/main/java/com/vultisig/wallet/ui/components/v2/topbar/V2Topbar.kt

```diff
@@ -0,0 +1,120 @@
+package com.vultisig.wallet.ui.components.v2.topbar
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.foundation.layout.WindowInsets
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.CenterAlignedTopAppBar
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBarDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.vultisig.wallet.R
+import com.vultisig.wallet.ui.components.UiSpacer
+import com.vultisig.wallet.ui.components.v2.buttons.DesignType
+import com.vultisig.wallet.ui.components.v2.buttons.VsCircleButton
+import com.vultisig.wallet.ui.components.v2.buttons.VsCircleButtonSize
+import com.vultisig.wallet.ui.components.v2.buttons.VsCircleButtonType
+import com.vultisig.wallet.ui.theme.Theme
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+internal fun V2Topbar(
+    title: String?,
+    onBackClick: () -> Unit = {},
+) {
+    CenterAlignedTopAppBar(
+        modifier = Modifier,
+        title = {
+            Text(
+                text = title.orEmpty(),
+                style = Theme.brockmann.headings.title3,
+                color = Theme.colors.text.primary,
```

Comment:
@aminsato please lets deprecated colors v1, and be consistance. or remove v2 colors if its unnecesary